public/js/extraFunctions.js

```diff
@@ -654,6 +654,143 @@ function tableData_closeAllSelect(elmnt) {
 
 }
 
+let initialize_resize_hamburger = function() {
+
+  //total width of all items in tab minus hamburger
+  let total_width = -44.25;
+
+  //width of all items to the left (because they get removed last) plus logout button and hamburger widths
+  //44.25 is hamburger width, has to be hardcoded becuase it might be display: none;'d
+  let left_width = 44.25 + $('#logout_button').outerWidth()
+
+  //gets all non-tablinks-right elements and adds their width to total_width  and also left_width
+  $('.tab .tablinks:not(.tablinks-right)').outerWidth(function(_i, w) {total_width += w; left_width += w;});
```

Comment:
Use arrow function syntax and don't make overlong (>80 chars) lines; also, there is no need to give a name to the throwaway argument (`_` is fine):
```suggestion
  $('.tab .tablinks:not(.tablinks-right)').outerWidth((_, w) => {
    total_width += w;
    left_width += w;
  });
```